test/test_tabulate.jl

```diff
@@ -0,0 +1,38 @@
+path = Pkg.dir("Dolo")
+
+Pkg.build("QuantEcon")
+import Dolo
+using DataArrays
```

Comment:
I don't think we directly use `DataArrays`, `DataFrames`, RDatasets`, or `DataStructures` in this file -- we shouldn't import them as it might cause issues if we decide to have CI run these tests.